python/ray/tests/test_gpu_objects_gloo.py

```diff
@@ -125,6 +125,64 @@ def test_gc_del_ref_before_recv_finish(ray_start_regular, data_size_bytes):
     )
 
 
+@pytest.mark.parametrize("data_size_bytes", [100])
```

Comment:
data_size_bytes doesn't appear to be used?

Reply:
good catch!

Reply:
removed